### DISCUSSION TITLE
Build on PyPI outdated

### DISCUSSION THREAD

**Discussion by elixirs-facing** [openai/whisper#1529] [General]
Hi,

the latest build on PyPI is 20230314. there seem to be a bunch of improvements for hallucinations in the comits which are not pushed to PyPI because no further tags were created in the last four months. is `main`  unstable and yet to be released, or has installation via PyPI been discontinued?

**Comment by elixirs-facing** [comment]
any updates on this would be appreciated

**Comment by MatejBabis** [comment]
Hi, I'd also like to get an update on this 🙏

**Comment by jongwook** [comment]
Just pushed a build [20230918](https://pypi.org/project/openai-whisper/20230918/) - hopefully it's not more unstable than usual..

**Reply by elixirs-facing** [reply]
perfect, thank you very much!
